android/app/build.gradle

```diff
@@ -72,7 +72,15 @@ import com.android.build.OutputFile
  * ]
  */
 
+  project.ext.vectoricons = [
+    // Name of the font files you want to copy
+    iconFontNames: [
+      'FontAwesome.ttf',
+      'Octicons.ttf']
```

Comment:
@machour The font `material` is required, because [RNE](https://react-native-training.github.io/react-native-elements/API/icons/#icon-props) use it as default values. I used to remove it, but got a crash in iOS simulator.

If we really do not want it, maybe another PR is required to set all icon type as `octicon` or something explicitly.